### ISSUE TITLE
What to delete (if anything) before re-training same model with a different data split?

### ISSUE THREAD

**Issue by davidkvcs** [MIC-DKFZ/nnUNet#954]
Hi 

I am training with a custom 5-fold datasplit.

After copying my preprocessed data to a local nvme-drive (to optimize training speed), I forgot to copy my custom splits_final.pkl file along to the local drive, a random one was created instead. I only noticed this, when I got an error using `nnUNet_find_best_configuration`.

I now have the following trained, 5 folds of each:
2d - on custom split
3d_lowres - on custom split
3d_fullres - on random split
3d_cascade_fullres - on random split (_didn't notice any errors or warning on this even though a different split was used here than on the "3d_lowres"-model_).

Of course, due to the above I am now getting an error when I now do `nnUNet_find_best_configuration`, since this needs npz-files to match across models.

I am now running 3d_fullres and 3d_cascade_fullres again using my custom split.

I did the first fold of 3d_fullres on my custom split now and as far as I can see it is populated with .npz-files for the random split as well as my custom split.

**Question**:  Do I need to delete old .npz-files (from the random split)? Other things I need to delete before re-training? (Assuming it will just overwrite old models with the new ones).

Thank you in advance!

Best regards
David

**Comment by davidkvcs** [comment]
Update: 
For now I just added the folders 

nnUNet_trained_models/nnUNet/3d_fullres/Task500_HNC01/nnUNetTrainerV2__nnUNetPlansv2.1
nnUNet_trained_models/nnUNet/3d_cascade_fullres/Task500_HNC01/nnUNetTrainerV2CascadeFullRes__nnUNetPlansv2.1

to a new folder called "old" and restarted the training with the correct splits.pkl file. 
Think that will work. Takes up a lot of space but should be a safe solution, I think.

**Comment by FabianIsensee** [comment]
Yes you will have to delete all the old training outputs if you are changing the folds.
Best,
Fabian

**Comment by Kobalt93** [comment]
Has your issue been resolved?
